doc/classes/Input.xml

```diff
@@ -198,6 +198,7 @@
 				If [param exact_match] is [code]false[/code], it ignores additional input modifiers for [InputEventKey] and [InputEventMouseButton] events, and the direction for [InputEventJoypadMotion] events.
 				[b]Note:[/b] Returning [code]true[/code] does not imply that the action is [i]still[/i] pressed. An action can be pressed and released again rapidly, and [code]true[/code] will still be returned so as not to miss input.
 				[b]Note:[/b] Due to keyboard ghosting, [method is_action_just_pressed] may return [code]false[/code] even if one of the action's keys is pressed. See [url=$DOCS_URL/tutorials/inputs/input_examples.html#keyboard-events]Input examples[/url] in the documentation for more information.
+				[b]Note:[/b] During input-handling (e.g. [method Node._input]) use [method InputEvent.is_action_pressed] instead, to query the action-state of the current event.
```

Comment:
```suggestion
				[b]Note:[/b] During input-handling (e.g. [method Node._input]), use [method InputEvent.is_action_pressed] instead to query the action-state of the current event.
```
Sentence flows better this way IMO

Reply:
I would suggest further:
```suggestion
				[b]Note:[/b] During input handling (e.g. [method Node._input]), use [method InputEvent.is_action_pressed] instead, to query the action state of the current event.
```

Same below. English doesn't hyphenate words as often as German would compound then :)